README.md

```diff
@@ -32,7 +32,7 @@ The hierarchy of content is diagramed here:
 ## Examples
 Two examples are provided of content documents that validate against the Washington Post ANS schema:
 
-1. [An Example Story](src/test/resources/com/washingtonpost/arc/ans/v0_3/model/story-fixture-tiny-house.json)
+1. [An Example Story](src/test/resources/com/washingtonpost/arc/ans/v0_3/model/collection-fixture-tiny-house.json)
```

Comment:
Did you actually change the name of the file?  I don't think you did, so maybe just keep it as "story-fixture-tiny-house.json"?